src/main/java/org/sonar/plugins/cas/util/JakartaHttpRequestAttributeWrapper.java

```diff
@@ -0,0 +1,33 @@
+package org.sonar.plugins.cas.util;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.Method;
+import org.sonar.api.server.http.HttpRequest;
+
+/**
+ * Wrapper for JakartaHttpRequest
+ * due to Sonar-Server Api restrictions the Attributes are no longer accessible
+ * The JakartaHttpRequestAttributeWrapper take use of reflections to access known fields
+ *
+ * @author Marco Bergen
+ */
+public class JakartaHttpRequestAttributeWrapper {
```

Comment:
```suggestion
public class HttpRequestAttributeWrapper {
```
is the Jakarta Prefix required or preferred?
I mean Jakarta itself does not mark their classes as such, do they?

Reply:
The real implementation wich is passed to the doAuthenticate Method at runtime is of type JakartaHttpRequest, therefor the wrapper was named this way. But to be more generic for later on, i'm going to rename it.